app/src/main/AndroidManifest.xml

```diff
@@ -884,6 +884,24 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".ExampleOverviewActivity" />
         </activity>
+        <activity
+            android:name=".examples.MultiDisplayActivity"
+            android:description="@string/description_multiple_display"
+            android:exported="true"
+            android:label="@string/activity_multiple_display">
+            <meta-data
+                android:name="@string/category"
+                android:value="@string/category_lab" />
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value=".ExampleOverviewActivity" />
+        </activity>
+        <activity
+            android:name=".examples.SecondaryDisplaySimpleMapActivity"
+            android:launchMode="singleTask">
```

Comment:
Just curious: is singleTask mandatory here?

Reply:
nit: could also explicitly make that activity `exported = false` (just because we can!)

Reply:
This activity is meant to be only launched in the secondary display, so won't export it, we can more think this activity as part of the MultiDisplayActivity.

Reply:
And yes, the singleTask is needed to be launched on the secondary display.